examples/gameserverallocation-deprecated.yaml

```diff
@@ -37,11 +37,6 @@ spec:
       game: my-game
     matchExpressions:
       - {key: tier, operator: In, values: [cache]}
-    # [Stage:Beta]
-    # [FeatureFlag:StateAllocationFilter]
-    # Specifies which State is the filter to be used when attempting to retrieve a GameServer
```

Comment:
Please leave in the description, but you can remove the lines:

```
    # [Stage:Beta]
    # [FeatureFlag:StateAllocationFilter]
```

Reply:
Included the description.